js/app.js

```diff
@@ -1,2 +1,44 @@
 $(function () {
+  $("#search").submit(() =>{
```

Comment:
@cbrooks2019 nice job implementing a `.submit()` listener here. However, best practice is to pass the `event` object as a parameter. `.submit((event) => {..})` - 

here an actual example from the solution: https://github.com/jsd20191008/weather-api-app-solution/blob/master/js/main-jquery-ajax.js#L2